### DISCUSSION TITLE
how can i add extension to the /scrape api?

### DISCUSSION THREAD

**Discussion by anispwyn** [browserless/chrome#4000] [Q&A] [answered]
Lets say, i want to scrape some site with bypass paywall clean (currently the devs got dmca'd), how can i do that?

I've read some docs and still dont understand some

Thanks for helping

**Comment by joelgriffith** [comment]
Are you needing extensions as in Chrome Extensions, or extending the functionality in our repository?

**Reply by anispwyn** [reply]
Chrome extension

**Reply by joelgriffith** [reply, marked as answer]
The best way to do so is to mount the extension as a volume inside of docker, then use the appropriate command line flags when launching to load that extension. Let me know if you need more help with any of that.
